Patches/python.diff

```diff
@@ -211,50 +205,9 @@ diff -r 8667c26e2bec Objects/codeobject.c
  /* Use co_lnotab to compute the line number from a bytecode index, addrq.  See
     lnotab_notes.txt for the details of the lnotab representation.
  */
-diff -r 8667c26e2bec Objects/object.c
---- a/Objects/object.c	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Objects/object.c	Mon Aug 31 14:44:51 2015 -0700
-@@ -1787,12 +1787,12 @@
- _Py_PrintReferences(FILE *fp)
- {
-     PyObject *op;
--    fprintf(fp, "Remaining objects:\n");
-+    fprintf(fp, "Remaining objects:\r\n");
-     for (op = refchain._ob_next; op != &refchain; op = op->_ob_next) {
--        fprintf(fp, "%p [%" PY_FORMAT_SIZE_T "d] ", op, op->ob_refcnt);
-+        fprintf(fp, "[%" PY_FORMAT_SIZE_T "d] ", op->ob_refcnt);
-         if (PyObject_Print(op, fp, 0) != 0)
-             PyErr_Clear();
--        putc('\n', fp);
-+        putc('\r\n', fp);
-     }
- }
- 
-diff -r 8667c26e2bec Objects/obmalloc.c
---- a/Objects/obmalloc.c	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Objects/obmalloc.c	Mon Aug 31 14:44:51 2015 -0700
-@@ -1175,6 +1175,7 @@
-     uint size;
- 
-     _Py_AllocatedBlocks++;
-+	//printf("Allocated: %d\r\n", _Py_AllocatedBlocks);
- 
-     assert(nelem <= PY_SSIZE_T_MAX / elsize);
-     nbytes = nelem * elsize;
-diff -r 8667c26e2bec Objects/typeobject.c
---- a/Objects/typeobject.c	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Objects/typeobject.c	Mon Aug 31 14:44:51 2015 -0700
-@@ -7208,6 +7208,7 @@
-             return -1;
-         }
-         if (co->co_argcount == 0) {
-+			assert(0);
```

Comment:
I had added that previously to try and track down some issues, again I'm removing my changes.